bundle.yaml

```diff
@@ -469,41 +469,39 @@ spec:
+          value: registry.connect.redhat.com/sumologic/kube-state-metrics@sha256:bb1f73b2be45fa31dfd3fbe72df1d553e5c6cdaa81709bf628764f5969ce4fc2
         - name: RELATED_IMAGE_PROMETHEUS_OPERATOR
-          value: public.ecr.aws/sumologic/prometheus-operator@sha256:8e29f9b5b7ae76aa9515d1ba20180511feb0c642bc61f782d8d1ea6f62f862aa
+          value: registry.connect.redhat.com/sumologic/prometheus-operator@sha256:8e29f9b5b7ae76aa9515d1ba20180511feb0c642bc61f782d8d1ea6f62f862aa
         - name: RELATED_IMAGE_PROMETHEUS_CONFIG_RELOADER
-          value: public.ecr.aws/sumologic/prometheus-config-reloader@sha256:0309a748632654700e0d49f7ecd7ee61e6344f53107c9ce0c5cf1d301f943d37
+          value: registry.connect.redhat.com/sumologic/prometheus-config-reloader@sha256:0309a748632654700e0d49f7ecd7ee61e6344f53107c9ce0c5cf1d301f943d37
         - name: RELATED_IMAGE_PROMETHEUS
-          value: public.ecr.aws/sumologic/prometheus@sha256:716d5c7183affd306bee7c35c577e5da852d4dfe8da9bf37aff255cd96a0c9b3
+          value: registry.connect.redhat.com/sumologic/prometheus@sha256:716d5c7183affd306bee7c35c577e5da852d4dfe8da9bf37aff255cd96a0c9b3
         - name: RELATED_IMAGE_THANOS
-          value: public.ecr.aws/sumologic/thanos@sha256:707e40ee919e4bc3c2edeecdc536acdf8cb67f9e5fc34eaad9d9e4c283438bb1
+          value: registry.connect.redhat.com/sumologic/thanos@sha256:707e40ee919e4bc3c2edeecdc536acdf8cb67f9e5fc34eaad9d9e4c283438bb1
         - name: RELATED_IMAGE_METRICS_SERVER
-          value: public.ecr.aws/sumologic/metrics-server@sha256:1f3c4554e2141e67600ffb2528d8745daf1e581330cb5f4fc75ee7c8aa2298c1
+          value: registry.connect.redhat.com/sumologic/metrics-server@sha256:1f3c4554e2141e67600ffb2528d8745daf1e581330cb5f4fc75ee7c8aa2298c1
         - name: RELATED_IMAGE_KUBE_RBAC_PROXY
-          value: public.ecr.aws/sumologic/kube-rbac-proxy@sha256:f92b5704d9d1290b1dd9cc8bfcc445e3e118794e8cf9531f439994ee4d2523a6
+          value: registry.connect.redhat.com/sumologic/kube-rbac-proxy@sha256:f92b5704d9d1290b1dd9cc8bfcc445e3e118794e8cf9531f439994ee4d2523a6
         - name: RELATED_IMAGE_TAILING_SIDECAR_OPERATOR
-          value: ghcr.io/sumologic/tailing-sidecar-operator@sha256:1d9d902b165596e12a626f5dd2f9d512627b356c3773a04c96df24270a840724
+          value: registry.connect.redhat.com/sumologic/tailing-sidecar-operator@sha256:1d9d902b165596e12a626f5dd2f9d512627b356c3773a04c96df24270a840724
         - name: RELATED_IMAGE_TAILING_SIDECAR
-          value: ghcr.io/sumologic/tailing-sidecar@sha256:838401ab699ae1f4a581a62a6eea413bca1e2c30d60358acb16bd86be75fbca1
+          value: registry.connect.redhat.com/sumologic/tailing-sidecar@sha256:838401ab699ae1f4a581a62a6eea413bca1e2c30d60358acb16bd86be75fbca1
         - name: RELATED_IMAGE_TELEGRAF_OPERATOR
-          value: public.ecr.aws/sumologic/telegraf-operator-ubi@sha256:3a63153408ff6bb5d6294fcdedfa30b07a5023ca5f72dd63c6e4de31f6bf1c68
```

Comment:
So it's not `-ubi` anymore?

Reply:
It is ubi image but container registry for it is `registry.connect.redhat.com/sumologic/telegraf-operator`, in `public.ecr.aws` we created repository with name `telegraf-operator-ubi` because `tag` for this image is fixed in telegraf-operator helm chart.